pkg/utils/patch.go

```diff
@@ -3,8 +3,8 @@ package utils
 import (
 	"context"
 	"encoding/json"
-	crdv1alpha1 "github.com/kubernetes-csi/external-snapshotter/client/v8/apis/volumegroupsnapshot/v1alpha1"
 
+	crdv1alpha1 "github.com/kubernetes-csi/external-snapshotter/client/v8/apis/volumegroupsnapshot/v1alpha1"
```

Comment:
This change is not necessary?  Can you remove it?

Reply:
It's just a cleanup, grouping together all the internal imports. Do you want me to open a PR just for that, or we leave it as it is?

Reply:
Oh, I see what you are doing.  That's fine then.